magenta/models/music_vae/js/data.ts

```diff
@@ -109,25 +95,29 @@ export class DrumsConverter extends DataConverter{
       drumRoll.set(1, i, -1);
     }
     notes.forEach((note) => {
-      drumRoll.set(1, note.startStep, this.pitchToClass[note.pitch]);
-      drumRoll.set(0, note.startStep, -1);
+      drumRoll.set(1, note.quantizedStartStep, this.pitchToClass[note.pitch]);
+      drumRoll.set(0, note.quantizedStartStep, -1);
     });
     return drumRoll.toTensor() as dl.Tensor2D;
   }
 
   toNoteSequence(oh: dl.Tensor2D) {
-    const notes: Note[] = [];
+    const noteSequence = new NoteSequence();
```

Comment:
Just an FYI, if we ever switched to using reflection instead of statically generated classes (probably unlikely), then NoteSequence.create() has better compatibility. It probably doesn't matter, but that's why I used .create() in the core library.

https://github.com/dcodeIO/ProtoBuf.js/#pbts-for-typescript

Reply:
Done.